cmd/litcli/sessions.go

```diff
@@ -115,6 +138,8 @@ func parseSessionType(sessionType string) (litrpc.SessionType, error) {
 		return litrpc.SessionType_TYPE_MACAROON_ADMIN, nil
 	case "readonly":
 		return litrpc.SessionType_TYPE_MACAROON_READONLY, nil
+	case "custom":
```

Comment:
An itest with the custom session would be nice. Just to make sure the permissions actually work correctly.

Reply:
ah yes of course (completely slipped my mind)